### PR TITLE
Add Tealium

### PR DESCRIPTION
**Pull request by craiglondon**
Testable at
https://autoclubsouth.aaa.com/
http://www.ikea.com/us/en/catalog/categories/departments/bedroom/16284/

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -7511,6 +7511,15 @@
 			"icon": "Tealeaf.png",
 			"website": "www.tealeaf.com"
 		},
+		"Tealium": {
+			"cats": [
+				36
+			],
+			"env": "^TEALIUMENABLED$",
+			"icon": "Tealium.png",
+			"script": "\/\/tags\\.tiqcdn\\.com\/",
```
Comment (jvoisin) [inline src/apps.json]: There is no need to escape the `/` character.

Comment (craiglondon) [inline src/apps.json]: Re: Escaping forward slash
When I use Regexr, it throws an error if I don't escape forward slash, why does it require them to be escaped but Wappalyzer does not?
http://www.regexr.com
`//tags\\.tiqcdn\\.com/`

**Comment by craiglondon** [comment]
Strange... tealium.com is a valid website.

@jvoisin any thoughts?

```Validating apps.json...
Validating regular expressions...
/home/travis/build/AliasIO/Wappalyzer/bin/wappalyzer-validate-regex:84
		throw new Error('Invalid website URL\n' + app + ': ' + json.apps[app].website);
		^
Error: Invalid website URL
Tealium: tealium.com
    at Object.<anonymous> (/home/travis/build/AliasIO/Wappalyzer/bin/wappalyzer-validate-regex:84:9)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:425:7)
    at startup (bootstrap_node.js:146:9)
    at bootstrap_node.js:540:3
npm ERR! Test failed.  See above for more details.```

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**